raster/r.topidx/global.h

```diff
@@ -6,8 +6,10 @@
 #define	atbv(i,j)		atb[i][j]
 #define	is_cv_null(i,j)		Rast_is_d_null_value(&cv(i,j))
 #define	is_atbv_null(i,j)	Rast_is_d_null_value(&atbv(i,j))
+#define is_atbv_unprocessed(i,j) (atbv(i,j) == UNPROCESSED)
 
 #define	ZERO			0.0000001
+#define	UNPROCESSED		-1.7976931348623157e+308
```

Comment:
```suggestion
#define	UNPROCESSED		-DBL_MAX
```

Reply:
Please also add `#include <float.h>` at the top

`DBL_MAX` is defined in float.h